src/ImageSharp/Formats/Bmp/BmpImageFormatProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿// <copyright file="PngImageFormatProvider.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Formats
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using ImageSharp.PixelFormats;
+
+    /// <summary>
+    /// Detects gif file headers
+    /// </summary>
+    public class BmpImageFormatProvider : IImageFormatProvider
+    {
+        /// <inheritdoc/>
+        public void Configure(IImageFormatHost host)
+        {
+            var encoder = new BmpEncoder();
+            foreach (string mimeType in BmpConstants.MimeTypes)
```

Comment:
I've decided to leave them for a time being, doesn't make a huge difference either way and at least the current way its obvious where the static data relating to a format is.